### PR TITLE
ref: update DVCLive links

### PR DESCRIPTION
**Pull request by dberenbaum**
Fixes dvclive links

### CODE REVIEW

File: content/docs/command-reference/run.md [hunk 1]
```diff
@@ -278,7 +278,7 @@ $ dvc run -n second_stage './another_script.sh $MYENVVAR'
   tracked by DVC. Useful if you prefer to track it with Git.
 
 - `--live-no-summary` - deactivates DVCLive
-  [summary](/doc/dvclive/user-guide/quickstart#metrics-summary) generation.
+  [summary](doc/dvclive/api-reference/live/log#description) generation.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/run.md:281]:
```suggestion
  [summary](/doc/dvclive/api-reference/live/log#description) generation.
```

File: content/docs/command-reference/stage/add.md [hunk 2]
```diff
@@ -258,7 +258,7 @@ data science experiments.
   tracked by DVC. Useful if you prefer to track it with Git.
 
 - `--live-no-summary` - deactivates DVCLive
-  [summary](/doc/dvclive/user-guide/quickstart#metrics-summary) generation.
+  [summary](doc/dvclive/api-reference/live/log#description) generation.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/stage/add.md:261]:
```suggestion
  [summary](/doc/dvclive/api-reference/live/log#description) generation.
```

**Review by jorgeorpinel — COMMENTED** [review]
Missing `/` 😬

**Review by daavoo — COMMENTED** [review]
@iterative/websites Shouldn't this be caught by link checker?

File: content/docs/command-reference/run.md [hunk 3]
```diff
@@ -278,7 +278,7 @@ $ dvc run -n second_stage './another_script.sh $MYENVVAR'
   tracked by DVC. Useful if you prefer to track it with Git.
 
 - `--live-no-summary` - deactivates DVCLive
-  [summary](/doc/dvclive/user-guide/quickstart#metrics-summary) generation.
+  [summary](/doc/dvclive/api-reference/live/log#description) generation.
 
 - `--live-no-html` - deactivates DVCLive
   [HTML report](/doc/dvclive/user-guide/dvclive-with-dvc#html-report)
```
Comment (daavoo) [inline content/docs/command-reference/run.md:284]:
```suggestion
  [HTML report](/doc/dvclive/dvclive-with-dvc#html-report)
```

File: content/docs/command-reference/stage/add.md [hunk 4]
```diff
@@ -258,7 +258,7 @@ data science experiments.
   tracked by DVC. Useful if you prefer to track it with Git.
 
 - `--live-no-summary` - deactivates DVCLive
-  [summary](/doc/dvclive/user-guide/quickstart#metrics-summary) generation.
+  [summary](/doc/dvclive/api-reference/live/log#description) generation.
 
 - `--live-no-html` - deactivates DVCLive
   [HTML report](/doc/dvclive/user-guide/dvclive-with-dvc#html-report)
```
Comment (daavoo) [inline content/docs/command-reference/stage/add.md:264]:
```suggestion
  [HTML report](/doc/dvclive/dvclive-with-dvc#html-report)
```

**Pull request merged**